WebApp/public/scripts/signaling.js

```diff
@@ -79,11 +88,17 @@ export default class Signaling extends EventTarget {
       lastTimeRequest = Date.parse(res.headers.get('Date'));
 
       const data = await res.json();
-      const candidates = data.candidates.filter(v => v.connectionId = this.connectionId);
-
-      if (candidates.length > 0) {
-        for (let candidate of candidates[0].candidates) {
-          this.dispatchEvent(new CustomEvent('candidate', { detail: candidate }));
+      const candidatesList = data.candidates.filter(v => v.connectionId = this.connectionId);
```

Comment:
Is this correct? `v => v.connectionId = this.connectionId`

Reply:
oh, this is not correct. I fix it.